priv/templates/nimble_template/.credo.exs

```diff
@@ -158,8 +158,8 @@
         #
         {Credo.Check.Readability.StrictModuleLayout,
          [
-           order:
-             ~w/shortdoc moduledoc behaviour use import alias require module_attribute defstruct callback/a
+          order: ~w/shortdoc moduledoc behaviour use import alias require module_attribute defstruct callback public_fun private_fun/a,
+          ignore: ~w/callback_impl/a
```

Comment:
should we add this rule into the `template itself` too @byhbt https://github.com/nimblehq/elixir-templates/blob/develop/.credo.exs#L168